### PR TITLE
Internalization

### PR DESCRIPTION
**Pull request by eggrobin**
#602 in `ksp_plugin` and its tests.

### CODE REVIEW

File: ksp_plugin/interface.cpp [hunk 1]
```diff
@@ -43,14 +45,20 @@ using base::PushDeserializer;
...
 using geometry::Displacement;
 using geometry::RadiusLatitudeLongitude;
+using geometry::Vector;
+using geometry::Velocity;
 using ksp_plugin::AliceSun;
 using ksp_plugin::Barycentric;
 using ksp_plugin::Part;
 using ksp_plugin::World;
+using physics::FrameField;
+using physics::DegreesOfFreedom;
```
Comment (pleroy) [inline ksp_plugin/interface.cpp]: Alphabetize.

File: ksp_plugin/interface_iterator.cpp [hunk 2]
```diff
@@ -8,10 +8,12 @@
 #include "ksp_plugin/plugin.hpp"
 
 namespace principia {
+namespace interface {
 
-using ksp_plugin::World;
+using physics::DegreesOfFreedom;
```
Comment (pleroy) [inline ksp_plugin/interface_iterator.cpp]: Remove blank line and alphabetize the `using`.

File: ksp_plugin/physics_bubble.hpp [hunk 3]
```diff
@@ -18,14 +18,21 @@
...
 namespace principia {
+namespace ksp_plugin {
+namespace internal_physics_bubble {
 
 using base::not_null;
 using geometry::Displacement;
 using geometry::OrthogonalMap;
+using geometry::Vector;
+using geometry::Velocity;
+using geometry::Position;
```
Comment (pleroy) [inline ksp_plugin/physics_bubble.hpp]: Alphabetize.

File: ksp_plugin_test/interface_test.cpp [hunk 4]
```diff
@@ -23,16 +23,20 @@
...
+using base::make_not_null_unique;
 using base::PullSerializer;
 using base::PushDeserializer;
 using geometry::AngularVelocity;
 using geometry::Bivector;
 using geometry::Displacement;
+using geometry::OrthogonalMap;
 using geometry::Vector;
 using geometry::Velocity;
+using geometry::Rotation;
```
Comment (pleroy) [inline ksp_plugin_test/interface_test.cpp]: Alphabetize.

File: ksp_plugin_test/plugin_integration_test.cpp [hunk 5]
```diff
@@ -13,18 +13,26 @@
...
 using geometry::Permutation;
+using integrators::DormandElMikkawyPrince1986RKN434FM;
 using physics::KeplerianElements;
 using quantities::Abs;
+using quantities::Acceleration;
 using quantities::ArcTan;
 using quantities::Cos;
+using quantities::GravitationalParameter;
 using quantities::Pow;
+using quantities::Length;
```
Comment (pleroy) [inline ksp_plugin_test/plugin_integration_test.cpp]: Alphabetize.

File: ksp_plugin/interface.cpp (hunk 1 above)
Comment (eggrobin) [inline ksp_plugin/interface.cpp]: Done.

File: ksp_plugin/interface_iterator.cpp (hunk 2 above)
Comment (eggrobin) [inline ksp_plugin/interface_iterator.cpp]: Done.

File: ksp_plugin/physics_bubble.hpp (hunk 3 above)
Comment (eggrobin) [inline ksp_plugin/physics_bubble.hpp]: Done.

File: ksp_plugin_test/interface_test.cpp (hunk 4 above)
Comment (eggrobin) [inline ksp_plugin_test/interface_test.cpp]: Done.

File: ksp_plugin_test/plugin_integration_test.cpp (hunk 5 above)
Comment (eggrobin) [inline ksp_plugin_test/plugin_integration_test.cpp]: Done.

**Pull request merged**